source/learn/rosetta_stone.md

```diff
@@ -0,0 +1,1882 @@
+# Python Fortran Rosetta Stone
+
+Python with NumPy and Fortran are very similar in terms of
+expressiveness and features. This rosetta stone shows how to implement
+many common idioms in both languages side by side.
+
+## How to Execute Code Snippets
+
+Consider for example the following code snippets:
+::::::{grid} 1 1 2 2
```

Comment:
Can you check the margins. There seems too much vertical spacing between the code blocks, maybe we have to reduce the *my* value.